editor/frontend/src/ProjectLocation/ProjectGeolocation.ts

```diff
@@ -308,7 +308,7 @@ export class ProjectGeolocationPointTool extends PrimitiveTool {
     if (!deco.updateEcefLocation(origin, this._origin, Angle.createRadians(this.north)))
       return;
 
-    this.onReinitialize(); // Calls onRestartTool to exit...
+    void this.onReinitialize(); // Calls onRestartTool to exit...

```

Comment:
Does this make it so you don't need the floating promise eslint ignore? I see you use it a few times in the PR but wasn't sure the behavior.

Reply:
Yes, very non-obviously. See previous comments - explicitly disabling the lint rule would be much better. Unfortunately the lint rule suggests `void` as the auto-fix.

Reply:
Ah okay, missed that comment before. I agree, I don't like this as a fix at all either, very easy to overlook...

I'll resolve this one since you brought up the same point in the other one.